docker/reverseproxy/conf.d/ssl.conf

```diff
@@ -0,0 +1,10 @@
+ssl_certificate /etc/letsencrypt/ssl-cert.cer;
```

Comment:
I wondered how you got these into the nginx. When testing locally I mounted a volume and then it worked but how do you do it on the server?

Reply:
The same way - I mounted the same folder from the host into the acme.sh container and the nginx container so the certs are persisted on the host and are readable by nginx.

Reply:
Oh never mind. I delete the line and forgot to take it over. :sweat_smile: 
Now the volume mount is in the reverse proxy.